src/source/Crypto/Dtls_mbedtls.c

```diff
@@ -0,0 +1,739 @@
+#define LOG_CLASS "DTLS_mbedtls"
```

Comment:
I did differentiate this on purpose, because this is a "special case" since the source is filtered using this pattern, https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/565/files#diff-af3b638bc2a3e6c650974192a53c7291R231-R236.

This is just to let us aware that, the binary will have a mutual exclusion depending on the flag.